index.js

```diff
@@ -2,24 +2,24 @@
 const os = require('os');
 const fs = require('fs');
 
+const MS_RE = /microsoft/i;
+
 const isWsl = () => {
 	if (process.platform !== 'linux') {
 		return false;
 	}
 
-	if (os.release().includes('Microsoft')) {
+	if (os.release().match(MS_RE) !== null) {
```

Comment:
You don't need a regex. Just use toLowerCase and Array#includes.

Reply:
Yes, thought about that, If it makes more sense than ok.